### ISSUE TITLE
Audio sounds muffled when using LWJGL3 backend

### ISSUE THREAD

**Issue by hujasonx** [libgdx/libgdx#6784]
Please ensure you have given all the following requested information in your report.

#### Issue details
Played sound effects sound very strange / muffled when using LWJGL3 backend.

#### Reproduction steps/code
The bug can be reproduced on a nearly empty project:
```
public class MyGdxGame extends ApplicationAdapter {
	Sound testSound;
	
	@Override
	public void create () {
		testSound = Gdx.audio.newSound(Gdx.files.internal("pistol2.wav"));
		testSound.play(1);
	}
	
	@Override
	public void dispose () {
		testSound.dispose();
	}
}
```
In the above test, pistol2.wav is downloaded from https://freesound.org/people/michorvath/sounds/427592/ but I'm pretty sure it doesn't matter what file you use to repro.

#### Version of libGDX and/or relevant dependencies
1.10.1-SNAPSHOT

#### Please select the affected platforms
- [ ] Android
- [ ] iOS
- [ ] HTML/GWT
* Windows
- [ ] Linux
- [ ] macOS

#### Other
The sound file I am playing is mono 16bit pcm Wav data at 44100hz. It also works fine on the lwjgl2 backend. I'd describe the audio distortion as as if it's being passed through a telephone or some sort of band pass filter.

To folks debugging: I wonder if this bug may have context / be related: https://github.com/libgdx/libgdx/issues/6526 - (also by chance the only other bug I have filed to this project - I tend to do a lot of audio stuff I guess)

**Comment by Frosty-J** [comment]
That's peculiar. I've been facing issues with poor sound quality in libGDX. But seeing as you linked that issue, I tried 1.10.1-SNAPSHOT, which has now fixed this issue for me on Linux. Some spectrogram comparisons with white noise:

![](https://user-images.githubusercontent.com/60154347/151652886-53ef9db7-3264-46f4-bf34-bf2fcc42a036.png)

**Comment by hujasonx** [comment]
Hi there, i see there is a "need more info" label, can you elaborate?

If it helps, I am on windows 11, though I don't see why this issue wouldnt also occur on windows 10.

**Comment by crykn** [comment]
Hm, it's quite strange that the issue only happens with LWJGL 3 and not 2. The respective files are basically the same: [com/badlogic/gdx/backends/lwjgl/audio/Wav.java](https://github.com/libgdx/libgdx/blob/master/backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/audio/Wav.java), [com/badlogic/gdx/backends/lwjgl3/audio/Wav.java](https://github.com/libgdx/libgdx/blob/master/backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/audio/Wav.java)

**Comment by Frosty-J** [comment]
I don't believe it has anything to do with the audio decoding part, rather the actual playback part, but in that case I don't know why it would affect OP, as LWJGL3's audio issue has been fixed for me presumably in the linked issue. I've now tested Windows 10 and got the same results as on Linux. Possibly hardware-dependent?

**Comment by Frosty-J** [comment]
Okay, this is still a problem across both Windows and Linux, `Sound` and `Music`, etc. The issue is very apparent with the [Vine boom](https://pixabay.com/sound-effects/vine-boom-162668/) sound when directly compared - quieter, and still not bassy enough even if amplified to normal levels.

I've narrowed it down to only monaural sounds being affected, so an easy workaround if you don't need panning is to encode all sounds in stereo.
